README.adoc

```diff
@@ -336,23 +336,39 @@ You can re-trigger individually the two downstream jobs, release and packaging.
 ====
 
 === Security
-The security release follows the same process as the stable one except that it also promotes git commits from a private repository to the public one and also promotes maven artifacts from a private maven repository to the public one.
+The security release follows the same process as the stable one excepted that artifacts are published in private location. So we need promote git commits from a private repository to the public one then promotes maven artifacts from a private maven repository to the public one.
```

Comment:
```suggestion
The security release follows the same process as the stable one except that artifacts are published in private. So we need promote git commits from a private repository to the public one then promote maven artifacts from a private maven repository to the public one.
```

Reply:
Thanks, one day I'll be good at writing (english) documentation